README.md

```diff
@@ -81,6 +81,20 @@ GOOS=linux GOARCH=arm64 go build -ldflags "-s -w" -o xray cmd/tracing/daemon.go
 ```
 As of Aug 31, 2020, windows and darwin builds for ARM64 are not supported by `go build`.
 
+## Pulling X-Ray Daemon image from ECR Public Gallery
+####  Pull alpha tag from Public ECR Gallery
+``
+docker pull public.ecr.aws/e2w6v5y3/amazon/aws-xray-daemon:alpha
```

Comment:
```suggestion
docker pull public.ecr.aws/amazon/aws-xray-daemon:alpha
```

I thought public ECR has these sort of readable names?

Reply:
no unfortunately we still have to enter `registry_alias`

Reply:
When pushing I'm guessing, but even when pulling? I don't believe it, it makes ECR public a strange option to use.

Reply:
For example here, I don't see any weird string

https://gallery.ecr.aws/datadog/agent

Image name is
public.ecr.aws/datadog/agent:7-jmx